src/puya/awst/nodes.py

```diff
@@ -1184,6 +1373,10 @@ def accept(self, visitor: ExpressionVisitor[T]) -> T:
 
 @attrs.frozen
 class PuyaLibData:
+    """
+    Metadata for puya library functions.
```

Comment:
This one should perhaps remain undocumented - not sure why it's not `_PuyaLibData` but I presume there was a reason.